README.md

```diff
@@ -187,7 +186,84 @@ s/fmt
 s/test
 ```
 
+### Testing on a Live Repo
+
+Due to the nature of a GitHub bot, the testing relies largley on mocks.
+For testing to see if a given feature will work it is recommended to create a
+GitHub App and a testing GitHub repo.
+
+#### Create a GitHub App via <https://github.com/settings/apps/new>
+
+1. Configure the permissions as described in the setup instructions above.
+2. Add a homepage URL (anything works)
+3. Setup the webhook URL
+
+   You probably want to use something like [`ngrok`](https://ngrok.com) for
+   this. If you do use `ngrok`, you may also want to signup for an account
+   via the [`ngrok` website](https://ngrok.com) so that your `ngrok` url
+   for the webhook doesn't expire.
+
+   With can then run `ngrok` with the Kodiak's dev port provided.
+
+   ```
+   ngrok http 8000
+   ```
+
+   Now we can copy the **Forwarding** url into the GitHub app form.
+   Be sure to copy the one with `https`.
+
+   Then hit create.
+
+4. Now install the GitHub
+
+   Use the **Install** option in the sidebar for the GitHub App.
+
+   You will want to create a testing GitHub repo with a Kodiak config file
+   with the `app_id` option set to your GitHub app's id.
+
+   This allows for the production version of Kodiak to be setup on all repos,
+   while allowing the testing version to run on the configured repo.
```

Comment:
```suggestion
   while allowing the testing version to run on the configured repo. If the
   production version of Kodiak finds a non-matching app_id, it will ignore
   the repository, leaving your local version to handle it.
```